tests/rt.sh

```diff
@@ -531,10 +531,9 @@ mkdir ${LOG_DIR}
 if [[ $ROCOTO == true ]]; then
 
   ROCOTO_XML=${PATHRT}/rocoto_workflow.xml
-  ROCOTO_STATE=${PATHRT}/rocoto_workflow.state
   ROCOTO_DB=${PATHRT}/rocoto_workflow.db
 
-  rm -f $ROCOTO_XML $ROCOTO_DB $ROCOTO_STATE *_lock.db
```

Comment:
Why is $ROCOTO_STATE removed?

Reply:
@DusanJovic-NOAA it seems to me that these lines were not there until @SamuelTrahanNOAA 's PR [1195](https://github.com/ufs-community/ufs-weather-model/pull/1195).

Reply:
@bbakernoaa Please revert those changes. It is okay to make these changes at this point in the commit since they do not affect the regression test results.